### DISCUSSION TITLE
set `config.lsp.hover.enabled = false` in your **Noice** config.

### DISCUSSION THREAD

**Discussion by tonmoy998** [folke/noice.nvim#615] [Q&A]
Each time I open a html file.It appears.

`vim.lsp.handlers["textDocument/hover"]` has been overwritten by another plugin?

Either disable the other plugin or set `config.lsp.hover.enabled = false` in your **Noice** config.
  - plugin: nvim
  - file: /tmp/.mount_nvimBFoNFw/usr/share/nvim/runtime/lua/vim/lsp.lua
  - line: 2319

**Comment by max397574** [comment]
well that's a quite clear error message
the handler gets overwritten by a different plugin
so either find that plugin and disable it or set the value in the config as written in the error message depending on which hover you want to use

**Reply by tonmoy998** [reply]
How do I disbale this error message in noice?

**Reply by rwxsb** [reply]
modifying it as in the answer here solved the issue for me https://github.com/folke/noice.nvim/issues/575
